planetiler-custommap/src/main/resources/samples/shortbread.yml

```
@@ -63,6 +63,8 @@ layers:
     attributes:
     - key: kind
       type: match_value
+    - key: way_area # area in square meter (Mercator Projection)
+      value: "${ feature.area('km2') * 1000000.0 }"
```

The docs only mention km2. If m2 is also avalible, the following would maybe be more performant.

Due to this not being testable, I am going to default to the save way ^^

```suggestion
      value: "${ feature.area('m2') }"
```

---

> (see [tests](https://github.com/onthegomap/planetiler/blob/9a0aa4fa6eea54cc85db3be8356b6cb42e3f7e4d/planetiler-custommap/src/test/java/com/onthegomap/planetiler/custommap/ConfiguredFeatureTest.java#L1595-L1611))

They don't cover `m2`, that is why I was asking.
Will try the input geometry in the evening, thanks ^^

---

Ah good call, I double checked and this does work, added a test to verify in #1226